sklearn/model_selection/tests/test_split.py

```diff
@@ -299,7 +236,7 @@ def test_kfold_indices():
     # Check all indices are returned in the test folds
     X1 = np.ones(18)
     kf = KFold(3)
-    check_cv_coverage(kf, X1, y=None, groups=None, expected_n_splits=3)
+    check_cv_coverage(kf, X1, y=None, groups=None, expected_n_splits=None)
```

Comment:
expected_n_splits=3 does an additional check, and it would be good to keep it.

Reply:
Yeah, probably better remove the `None` default value for `expected_n_splits`. It's always called with an int, anyway.